From 329c2f6b2075713fe7dd2c626175745b9209ad0b Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 20 Dec 2019 10:54:42 +0100
Subject: [PATCH 4/5] ffmpeg: Add MPEG-2 video decoder

As should be used for DVD playback, rather than the older libmpeg2-based
one fro gst-plugins-ugly.

This video decoder also supports MPEG-1 video playback.
---
 elements/extensions/ffmpeg-full/ffmpeg.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/extensions/ffmpeg-full/ffmpeg.bst b/elements/extensions/ffmpeg-full/ffmpeg.bst
index 61c8cacb..bd1275f0 100644
--- a/elements/extensions/ffmpeg-full/ffmpeg.bst
+++ b/elements/extensions/ffmpeg-full/ffmpeg.bst
@@ -8,7 +8,7 @@ variables:
   extra-aud-enc: wmav1,wmav2
   extra-aud-dec: ape,dca,eac3,mlp,tak,truehd,wmav1,wmav2,wmapro
   extra-vid-enc: h263,h264,wmv1,wmv2
-  extra-vid-dec: cinepak,flv,hevc,h263,h264,indeo2,indeo3,indeo4,indeo5,mpeg4,msmpeg4,msmpeg4v1,msmpeg4v2,vp6,vp6a,vp6f,wmv1,wmv2,wmv3,wmv3image
+  extra-vid-dec: cinepak,flv,hevc,h263,h264,indeo2,indeo3,indeo4,indeo5,mpeg2video,mpeg4,msmpeg4,msmpeg4v1,msmpeg4v2,vp6,vp6a,vp6f,wmv1,wmv2,wmv3,wmv3image
   extra-hwaccels: h264_vaapi,h264_vdpau,hevc_vaapi,hevc_vdpau
   extra-parsers: hevc,h264,dca
   extra-demuxers: avi,h264,m4v
-- 
2.26.2

